crates/aptos/src/common/types.rs

```diff
@@ -77,6 +78,7 @@ impl CliError {
     }
 }
 
+/// Config saved to `.aptos/config.yaml`
```

Comment:
nit: is this bubbled up to the user? Also, is it not configurable? If not, we might want to make it more obvious than just putting it in code 😄

Reply:
Is the location bubbled up to the user?

For simplicity it's not configurable and there's no global variables, it's directory based (like terraform and others).

It does output where the config file is written to, but in debug logs. We can probably add a new message in the init flow.